cli-qpid-jms/src/main/resources/log4j.properties

```diff
@@ -34,8 +34,9 @@ log4j.additivity.com.redhat.mqe.jms.Statistics=false
 
 log4j.logger.io.netty=WARN, console
 log4j.logger.netty=WARN, console
-# log4j.logger.org.apache.qpid=WARN, console
```

Comment:
Can be useful in some cases to have it in code.

Reply:
I will keep it, then. Maybe add comment, This is what you uncomment to get moar logz

Reply:
done